src/main/java/botimusprime/tasks/Event.java

```diff
@@ -0,0 +1,42 @@
+package botimusprime.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+    public Event(String description, boolean isDone, LocalDateTime from, LocalDateTime to) {
+        super(description, isDone);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public boolean isEvent() {
+        return true;
+    }
+
+    @Override
+    public boolean isToDo() {
+        return false;
+    }
+
+    @Override
+    public boolean isDeadline() {
+        return false;
+    }
+
```

Comment:
You may add a Override tag here. Can do the same for other tasks that extends Task class too.